.github/workflows/build.yml

```diff
@@ -20,11 +20,19 @@ on:
 
 jobs:
   build-xcarchive:
-    uses: StanfordBDHG/.github/.github/workflows/archive.yml@v2
+    uses: StanfordBDHG/.github/.github/workflows/archive.yml@feat/enable-interop-archiving
```

Comment:
Note to ourselves: Adjust the checked out version as soon as https://github.com/StanfordBDHG/.github/pull/49 is merged